### PR TITLE
Added edit client details functionality

### PR DESCRIPTION
**Pull request by SagarVishal**

### CODE REVIEW

File: web/mixins/editClient.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    async editClient(data) {
+      await this.$store
+        .dispatch("updateClient", data)
+        .then(() => {
+          this.$toasted.success("Client updated successfully", {
+            theme: "bubble",
+            position: "top-right",
+            duration: 2000,
+          });
+          this.$nextTick(() => {
```
Comment (ekta-kakadia) [inline web/mixins/editClient.js]: why have you use nextTick here ? Any issue facing here

**Comment by ekta-kakadia** [comment]
Please upload video

File: web/mixins/editClient.js (hunk 1 above)
Comment (SagarVishal) [reply web/mixins/editClient.js]: @ekta-kakadia removed

**Comment by SagarVishal** [comment]
https://user-images.githubusercontent.com/59005440/111589477-e9d3c900-87ea-11eb-92dd-e17f3b11ecdf.mp4

**Review by ekta-kakadia — APPROVED** [review]

**Pull request merged**